core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -218,7 +218,7 @@ final class ShuffleBlockFetcherIterator(
   }
 
   private[this] def sendRequest(req: FetchRequest) {
-    logDebug("Sending request for %d blocks (%s) from %s".format(
+    logDebug("Sending request for %d blocks (%s) to %s".format(
```

Comment:
I think the statement is correct in that it's saying that it's requesting bytes _from_ the remote address. The request is sent _to_ that address, sure. "from" relates to "blocks" not "Sending".

Reply:
either way it's confusing, shall we change to 

Requesting for %d blocks (%s) from %s"

Reply:
(remove 'for')
I don't think that's materially different. I'd just leave it.

Reply:
Yea, let's don't fix. I don't think it virtually matters.